pages/create-bounty.tsx

```diff
@@ -649,57 +636,54 @@ export default function CreateBountyPage() {
       );
   }
 
-  function renderBountyToken(type: "bounty" | "reward") {
-    const fieldParams = {
-      bounty: {
-        token: transactionalToken,
-        setToken: setTransactionalToken,
-        default: transactionalToken,
-        decimals: transactionalERC20?.decimals,
-        amount: issueAmount,
-        setAmount: setIssueAmount,
-        tokens: customTokens.filter((token) => !!token?.network_tokens?.isTransactional),
-        balance: transactionalERC20.balance,
-        isFunding: isFundingType,
-        label: t("bounty:fields.select-token.bounty", {
-          set: t("bounty:fields.set"),
-        }),
-      },
-      reward: {
-        token: rewardToken,
-        setToken: setRewardToken,
-        default: rewardToken,
-        decimals: transactionalERC20?.decimals,
-        amount: rewardAmount,
-        setAmount: setRewardAmount,
-        tokens: customTokens.filter((token) => !!token?.network_tokens?.isReward),
-        balance: rewardERC20.balance,
-        isFunding: isFundingType,
-        label: t("bounty:fields.select-token.reward", {
-          set: t("bounty:fields.set"),
-        }),
-      },
-    };
-
+  function renderButtons() {
     return (
       <>
-        <CreateBountyTokenAmount
-          currentToken={fieldParams[type].token}
-          setCurrentToken={fieldParams[type].setToken}
-          customTokens={fieldParams[type].tokens}
-          userAddress={currentUser?.walletAddress}
-          defaultToken={fieldParams[type].default}
-          canAddCustomToken={false}
-          addToken={addToken}
-          decimals={fieldParams[type].decimals}
-          issueAmount={fieldParams[type].amount}
-          setIssueAmount={fieldParams[type].setAmount}
-          tokenBalance={fieldParams[type].balance}
-          isFunders={type === "reward" ? false : true}
-          needValueValidation={!isFundingType || type === "reward"}
-          isFunding={isFundingType}
-          labelSelect={fieldParams[type].label}
-        />
+        <div className="col-6 ps-2">
+          <Button
+            className="col-12 bounty-outline-button"
+            onClick={() => {
+              currentSection !== 0 &&
+                setCurrentSection((prevState) => prevState - 1);
+            }}
```

Comment:
I think that this callback should be declared out of the render method. And I think that using `&&` operator for this purpose make the code hard to understand, as usually it is used when we want to get the boolean result. If all we need is check a condition and execute a code, use `if`.